### ISSUE TITLE
Can't get to IMAGE dropdown menu/button on screen readers

### ISSUE THREAD

**Issue by Cybernide** [Shared-Reality-Lab/IMAGE-browser#420]
OS: Windows
Browser: Chrome (Production version) and Opera (Test version)
Screenreader: NVDA and JAWS
Upon loading a website, the invisible 'Interpret this photo with IMAGE' **button** is **accessible** (either through tabbing or ) to screen reader after drop-down menu for developer mode and Monarch options ON MOST PAGES (tested pages: IMAGE homepage, a Google Images search for 'crab'). The invisible **drop-down menu** is **inaccessible** in this case.
When I tab away from the page and tab back to it, the invisible 'Interpret this photo with IMAGE' **button** is then **inaccessible** via screen reader, and the **drop-down menu** suddenly becomes **accessible** to the screen reader. After this point, the button is inaccessible until I reload the page.

This doesn't seem to happen when I switch applications, only when I switch between browser tabs.

The only exception to this seems to be the tutorial.html page. I cannot seem to get the buttons on this page.

**Comment by Cybernide** [comment]
Assigned to @jeffbl for triage
